pkgs/applications/kde/grantleetheme/grantlee-merge-theme-dirs.patch

```diff
@@ -44,17 +44,6 @@ Index: grantleetheme-17.04.0/src/grantleetheme.cpp
      loader->setTheme(dirName);
  
      if (!sEngine) {
-@@ -102,9 +105,7 @@ QString ThemePrivate::errorTemplate(cons
```

Comment:
Why is this patch changing?

Reply:
The purpose of this part is unclear to me. The patch as it was was reverting an improvement, namely the check for a null pointer.